dekalee/pubsub-swarrot-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "Dekalee\\PubSubSwarrotBundle\\DekaleePubSubSwarrotBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "DEKALEE_PUBSUB_SWARROT_KEY_FILE_PATH": "your-pub_sub-key-file-path"
```

Comment:
This should be in a comment to allow users use `bin/console secrets:set` command. 

See https://github.com/symfony/recipes-contrib/pull/785

Reply:
thanks for the tip, I have updated the PR